docs/themes.md

```diff
@@ -4,43 +4,42 @@ By default, blocks provide their styles to enable basic support for blocks in th
 
 Some advanced block features require opt-in support in the theme itself as it's difficult for the block to provide these styles, they may require some architecting of the theme itself, in order to work well.
 
-To opt-in for one of these features, we should call `add_theme_support( 'gutenberg', $features )` in the `functions.php` file of the theme. For example:
+To opt-in for one of these features, we should call `add_theme_support( 'feature-name', $optional-parameters )` in the `functions.php` file of the theme. For example:
```

Comment:
Noting that this is a syntax error on `$optional-parameters` (dashes aren't valid in variable name). I think we might want to drop the arguments here since they're a bit misleading (reader might not interpret `'feature-name'` as dummy text).

>To opt in for one of these features, we should call `add_theme_support` in the `functions.php` file of the theme:

Reply:
Thanks for pointing that out! Wasn't sure what was optimal here.

Good suggestion, especially since there's a full example immediately below. Added a commit to make this change. I also removed the "we" wording, since it didn't seem to be used in other places in this doc, and it was a little confusing who it was referring to. If that's not wanted, I can revert it.